src/actions/roundCorners.ts

```diff
@@ -6,6 +6,26 @@
  * {@link https://cloudinary.com/documentation/video_manipulation_and_delivery#rounding_corners_and_creating_circular_videos | Rounded videos}
  * @memberOf Actions
  * @namespace RoundCorners
+ * @example
+ * <caption>Round corners by a radius</caption>
+ * import {Cloudinary} from "@cloudinary/base";
+ * import {byRadius} from "@cloudinary/base/actions/roundCorners";
+ *
+ * const yourCldInstance = new Cloudinary({cloud:{cloudName:'demo'}});
+ * const image = yourCldInstance.image('woman');
+ * image.roundCorners(byRadius(10));
+ * image.toURL();
+ *
+ * @example
+ * <caption>Maximum rounded corners</caption>
+ * import {Cloudinary} from "@cloudinary/base";
+ * import {max} from "@cloudinary/base/actions/roundCorners";
+ *
+ * const yourCldInstance = new Cloudinary({cloud:{cloudName:'demo'}});
+ * const image = yourCldInstance.image('woman');
+ *
+ * image.roundCorners(max());
```

Comment:
Lets make these examples shorted by applying two actions on the same image in a single example

Reply:
Done